nixos/modules/services/continuous-integration/hydra/default.nix

```diff
@@ -37,36 +37,6 @@ let
 
   haveLocalDB = cfg.dbi == localDB;
 
-  hydra-package =
-  let
-    makeWrapperArgs = concatStringsSep " " (mapAttrsToList (key: value: "--set \"${key}\" \"${value}\"") hydraEnv);
```

Comment:
`--set-default` will allow it to work outside a service and inside.

Reply:
Thanks, that's a much cleaner solution!